appinfo/info.xml

```diff
@@ -29,8 +29,8 @@
 	<repository type="git">https://github.com/nextcloud/contacts.git</repository>
 	<screenshot>https://raw.githubusercontent.com/nextcloud/screenshots/master/apps/Contacts/contacts.png</screenshot>
 	<dependencies>
-		<owncloud min-version="9.0" max-version="9.2" />
-		<nextcloud min-version="9" max-version="13" />
+		<owncloud min-version="9.2" />
+		<nextcloud min-version="11" max-version="13" />
 	</dependencies>
 	<ocsid>168708</ocsid>
```

Comment:
@MariusBluem Do you have a doc where all of those are written? I tried to find it but the doc don't seemed to be up-to-date :'(